zebra-scanner/Cargo.toml

```
@@ -0,0 +1,44 @@
+[package]
+name = "zebra-scanner"
+version = "1.0.0-beta.30"
+authors = ["Zcash Foundation <zebra@zfnd.org>"]
+description = "Blockchain scanner for the Zebra Zcash node"
+license = "MIT OR Apache-2.0"
+repository = "https://github.com/ZcashFoundation/zebra"
+edition = "2021"
+
+readme = "../README.md"
+homepage = "https://zfnd.org/zebra/"
+# crates.io is limited to 5 keywords and categories
+keywords = ["zebra", "zcash"]
+# Must be one of <https://crates.io/category_slugs>
+categories = ["cryptography::cryptocurrencies"]
+
+[features]
+
+# Production features that activate extra dependencies, or extra features in dependencies
+
+[dependencies]
+zcash_client_backend = "0.10.0-rc.1"
+zcash_primitives = "=0.13.0-rc.1"
```

This strict version requirement doesn't match our other crates. You could add a comment explaining why it's needed, or change it to match.

```suggestion
zcash_primitives = "0.13.0-rc.1"
```

---

This is a typo, no need to be the strict version. Fixed in https://github.com/ZcashFoundation/zebra/pull/7758/commits/2ca1c93ce72456d06e0d2ca26ed7bcff0f946522